content/guides/outliers.md

```diff
@@ -19,7 +19,7 @@ Outlier Detection is an algorithmic feature that allows you to detect when some
 
 ## How to Use Outlier Detection on Your Data
 
-We’ve added a new query function called `outliers` to our query language. This function will return the usual results but outlier series will be marked.
+We’ve added a query function called `outliers` to our query language. This function will return the usual results but outlier series will be marked.
```

Comment:
i don't think we should have language like 'we have added...'. This could be in the docs for a long time and there are no dates on the docs, so when someone reads it in 2 years, did we just do it last week? Just describe the function.